data/hai/hai reveal 4 middle.txt

```diff
@@ -1510,60 +1913,131 @@ mission "Hai Reveal [A11] Sound the Alarm"
 			`	One Elder cries out, "You wouldn't dare!" To which Teeneep responds, "It is already scheduled to happen unless I actively stop it!"`
-			`	Before any further exchange can occur, though, the moderator bellows for security to escort you all from the session and shouts down the offending Elder for impropriety. The two remain engaged in a lively shouting match as you depart, the echoes following you until the chamber doors are shut.`
-			`	Teeneep moves to pull out a datapad, but both the security officers and Sayari step in to stay her hand. "The chamber will have to vote on the proposition to act," says Sayari, looking at Teeneep pointedly. "You know that external contact is forbidden during the deliberation grace window."`
+			`	Before any further exchange can occur, the moderator bellows for security. Guards escort you, Teeneep, Sayari, and the functionary, from the session. The moderator shouts down the offending Elder for impropriety and the two engage in a lively shouting match. Its echoes follow you until the chamber doors are shut.`
+
+
+			# Proceedings end.
+			action
+				"Hai Reveal [A11] Sound the Alarm: time" = 7
+			`	Teeneep moves to pull out a datapad, but both the security officers and Sayari step in to stay her hand. "The chamber will have to vote on the proposition to act," says Sayari. She looks at Teeneep pointedly. "You know that external contact is forbidden during the deliberation grace window."`
 			choice
 				`	"The deliberation grace window?"`
-				`	"So what, do we just wait?"`
+				`	"Just wait? I'll wait in the reading room."`
+					goto reading
+				`	"Why are we waiting?"`
 					goto wait
-			`	Teeneep sighs and points an exasperated expression your way. "Yes, after certain types of chamber hearings there is an enforced period where the petitioner must remain isolated to the chamber's waiting area, including digitally. I forgot for a moment."`
+			`	Teeneep sighs and points an exasperated expression your way. "Yes, after certain types of chamber hearings there is an enforced period. The petitioner must be secluded in the chamber's waiting area. I forgot for a moment."`
 			label wait
 			`	Sayari smiles at you. "Don't worry though, it's only for about twenty-three of your minutes." She eyes the security officer warily. "After a session such as that though, the traditional wisdom is to wait in relative silence."`
 			`	"If it wasn't a closed session we wouldn't be doing this," harrumphs Teeneep. "I hadn't realized it would be so didn't plan for it."`
```

Comment:
```suggestion
			`	"If it wasn't a closed session we wouldn't be doing this," harrumphs Teeneep. "I hadn't realized it would be this way and so didn't plan for it."`
```